### PR TITLE
added link to wiki

### PR DESCRIPTION
**Pull request by r-a-v-a-s**
This launches the default browser and goes to our wiki.
The links under Help → Manual do not work.

### CODE REVIEW

File: librecad/src/main/qc_applicationwindow.cpp [hunk 1]
```diff
@@ -3664,3 +3670,8 @@ void QC_ApplicationWindow::updateWindowTitle(QWidget *w)
         w->setWindowTitle(w->windowTitle()+m_qDraftModeTitle);
     }
 }
+
+void QC_ApplicationWindow::goto_wiki()
+{
+    QDesktopServices::openUrl(QUrl("http://wiki.librecad.org/index.php/Main_Page"));
```
Comment (lordofbikes) [inline librecad/src/main/qc_applicationwindow.cpp]: Keep the URL as short as possible:

`QUrl("http://wiki.librecad.org/")`

When we move the Wiki, the direct page link becomes invalid.

Comment (r-a-v-a-s) [inline librecad/src/main/qc_applicationwindow.cpp]: Any idea when the move will happen?

**Comment by r-a-v-a-s** [comment]
I couldn't resist the commit summary... partly because I have a 14 letter German last name. :innocent: :de:

**Comment by dxli** [comment]
is it okay for us to refer the user manual link to wiki as well?

File: librecad/src/main/qc_applicationwindow.cpp (hunk 1 above)
Comment (lordofbikes) [inline librecad/src/main/qc_applicationwindow.cpp]: Not yet, limited time and the heat here delays my progress on troubleshooting the content transfer tool.
Hope I can finish this in my holidays in August/September.

**Comment by lordofbikes** [comment]
I'd agree, linking to the Wiki is better than the empty manual now.
Or let us change the manual content and add the Wiki link there.

**Comment by r-a-v-a-s** [comment]
I think this is a sufficient temporary fix.

I want to talk about working on the help manual on the forum soon.

**Pull request merged**